client/src/components/ElementEditor/AddElementPopoverContent.js

```diff
@@ -0,0 +1,58 @@
+/* global window */
+
+import React, { Component, PropTypes } from 'react';
+import { Button } from 'reactstrap';
+import classNames from 'classnames';
+
+/**
+ * The AddElementPopoverContent component used in the context of an ElementEditor shows the
+ * available elements that can be added to an ElementalArea.
+ */
+class AddElementPopoverContent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.renderElementButtons = this.renderElementButtons.bind(this);
+  }
+
+  renderElementButtons() {
+    const { elementTypes } = this.props;
+
+    return elementTypes.map((elementType) =>
+      (
+        <Button className={
```

Comment:
You will need to add a `key=` in here. There is an identifier (of sorts) on the `elementType` you can use.